.github/workflows/types.yaml

```diff
@@ -5,6 +5,7 @@ on:
     branches:
       - "master"
       - "staging"
+      - "develop"
```

Comment:
is this correct? Cause the name of the branch is "dev", not "develop" (as in the backend)

Reply:
fixed